plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryMetadata.java

```diff
@@ -16,6 +16,7 @@
 import com.google.cloud.bigquery.BigQueryException;
 import com.google.cloud.bigquery.DatasetId;
 import com.google.cloud.bigquery.DatasetInfo;
+import com.google.cloud.bigquery.ExternalTableDefinition;
```

Comment:
Thank you for your contribution. We need a test in BaseBigQueryConnectorTest class.

Reply:
I added a test. PTAL

Reply:
The PR isn't updated.

Reply:
Sent it too soon. Updated.